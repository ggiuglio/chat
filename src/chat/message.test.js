import React from 'react';
import renderer from 'react-test-renderer';
import Message from './message';

describe('Message', () => {
  let component;
  const message = {
    author: 'Giulio',
    message: 'This is my message',
    date: '19 may 2020 18:34'
  }

  beforeEach(() => {
      component = renderer.create(
          <Message message = {message}/>
      );
  });

  it('should render a message', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });

});