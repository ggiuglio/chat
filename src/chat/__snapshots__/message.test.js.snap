// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render a message 1`] = `
<div
  className="sc-bdVaJa bVzWCR"
>
  <div
    className="sc-bwzfXH bGLOCH"
  >
    Giulio
  </div>
  <div
    className="sc-htpNat SYTas"
  >
    This is my message
  </div>
  <div
    className="sc-bxivhb dMhSOm"
  >
    19 may 2020 18:34
  </div>
</div>
`;
